packages/cozy-harvest-lib/src/components/TriggerLauncher.jsx

```diff
@@ -93,12 +102,19 @@ export class TriggerLauncher extends Component {
     if (typeof onSuccess === 'function') onSuccess(trigger)
   }
 
+  async handleLoginSuccess() {
+    this.dismissTwoFAModal()
```

Comment:
should we check if we have this modal displayed before dismissing it? It could save one render call

Reply:
I said 👍 but doe sit actually save a render call ? If the state is the same I don't think it will trigger an update ?

Reply:
It will call the render method and execute it. But it'll not re-render the dom if nothing change (but will call chidren render)

Reply:
To build on @Crash-- answer, TriggerLauncher is not a PureComponent so it does not have the shouldComponentUpdate check on props/state that would bypass the render if state/props had not changed.